object_service/tests/test_internals_nedsrv.py

```diff
@@ -46,9 +50,9 @@ def test_get_ned_objects(self):
     def test_get_ned_objects_unknown_object(self):
         '''Test to see if retrieval of NED objects method behaves as expected'''
         from object_service.NED import get_ned_data
-        identifiers = map(str, range(4))
+        identifiers = list(map(str, list(range(4))))
```

Comment:
I think this inner `list(...)` (automatically added by the porting tools) might not be needed here, although the outer `list(...)` might be needed.